### ISSUE TITLE
Extend getJsonResponse to include relationship properties

### ISSUE THREAD

**Issue by mdantonio** [rapydo/issues#67]
Now `getJsonResponse` (from `restapi/definitions.py`) follows relationships to add to the json the linked nodes

```
{
  "Meta": {
    "data_type": "<class 'list'>",
    "elements": 1,
    "errors": 0,
    "status": 200
  },
  "Response": {
    "data": [
      {
        [...]
        "relationships": {
          "languages": [
            {
              "attributes": {
                "code": "fr"
              },
              "id": "1657",
              "type": "language"
            },
    [...]
}
```

This operation totally skip properties defined on Relationships

for example -[languages {usage: audio}]->(LanguageNode)

**Comment by mdantonio** [comment]
getJsonResponse is now extended to retrieve fields from Relationship models

relationship properties are added to the attributes dictionary of the node. Since collisions are possible (in case of a Node model and a Relationship model both defining a property with the same name and show=True) i added a warning to report such cases.

Example:

`Name collision XYZ on node {'id': '314', 'type': 'creationlanguage', 'attributes': {'value': 'it'}} from both model <class 'imc.models.neo4j.CreationLanguage'> and property model <class 'imc.models.neo4j.Test'>`

This improvements will be included the in the next package (0.5.5?)

I wait for a feedback from IMC Team before closing this issue

**Comment by mdantonio** [comment]
> I wait for a feedback from IMC Team before closing this issue

It works
